### PR TITLE
Bump version to 12.1.1

### PR DESCRIPTION
**Pull request by maxxcrawford**
Bump version to `12.1.1`

- Blocked by #639 
  - The CHANGELOG entry  is updated in PR #639

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
-# HEAD
+# 12.1.1
 
 ## Features
 
 * **css:** Revise sticky promo component to be hidden by default  (#638)
 
-# 12.0.1
```
Comment (maxxcrawford) [inline CHANGELOG.md:7]: This was overwritten in #629 accidentally. Making it match the current/last release.

File: package-lock.json [hunk 2]
```diff
@@ -1505,25 +1505,29 @@
           "dependencies": {
             "abbrev": {
               "version": "1.1.1",
-              "bundled": true,
+              "resolved": false,
+              "integrity": "sha512-nne9/IiQ/hzIhY6pdDnbBtz7DjPTKrY00P/zvPSm5pOFkl6xuGrGnXn/VtTNNfNtAfZ9/1RtehkszU9qcTii0Q==",
```
Comment (craigcook) [inline package-lock.json:1509]: I don't think the rest of these changes to package-lock should be part of this PR, just the version number above (though we probably do need to update dependencies separately... it's been a while).

**Review by craigcook — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 2 above)
Comment (maxxcrawford) [reply package-lock.json]: Copy that. Just reverted it. 👍

**Review by craigcook — APPROVED** [review]
:shipit:

**Pull request merged**